<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>691</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FreeAdjust</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>2</horstretch>
     <verstretch>2</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidgetMain">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_network">
       <attribute name="title">
        <string>Сеть</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>0</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="handleWidth">
           <number>4</number>
          </property>
          <widget class="QTabWidget" name="tabWidgetNetwork">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Проект</string>
            </attribute>
            <widget class="QTextEdit" name="projectShortDescription">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>391</width>
               <height>151</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>170</y>
               <width>121</width>
               <height>16</height>
              </rect>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::NoContextMenu</enum>
             </property>
             <property name="text">
              <string>Предполагаемая С.К.О</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="sigma_apriory">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>170</y>
               <width>113</width>
               <height>20</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>200</y>
               <width>151</width>
               <height>16</height>
              </rect>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::NoContextMenu</enum>
             </property>
             <property name="text">
              <string>Доверительная вероятность</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="confidence_probability">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>200</y>
               <width>113</width>
               <height>20</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>230</y>
               <width>161</width>
               <height>16</height>
              </rect>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::NoContextMenu</enum>
             </property>
             <property name="text">
              <string>Допуск</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="tol_abs">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>230</y>
               <width>113</width>
               <height>20</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_4">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>260</y>
               <width>161</width>
               <height>16</height>
              </rect>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::NoContextMenu</enum>
             </property>
             <property name="text">
              <string>Используемое С.К.О</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_5">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>290</y>
               <width>161</width>
               <height>16</height>
              </rect>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::NoContextMenu</enum>
             </property>
             <property name="text">
              <string>Обновлять контрольные точки</string>
             </property>
            </widget>
            <widget class="QComboBox" name="sigma_act">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>260</y>
               <width>191</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QComboBox" name="update_cc">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>290</y>
               <width>191</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="Line" name="line">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>310</y>
               <width>381</width>
               <height>20</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
            <widget class="QLineEdit" name="stddev_direction">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>330</y>
               <width>113</width>
               <height>20</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_6">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>330</y>
               <width>161</width>
               <height>16</height>
              </rect>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::NoContextMenu</enum>
             </property>
             <property name="text">
              <string>С.К.О направлений</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="stddev_angle">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>360</y>
               <width>113</width>
               <height>20</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_7">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>360</y>
               <width>161</width>
               <height>16</height>
              </rect>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::NoContextMenu</enum>
             </property>
             <property name="text">
              <string>С.К.О горизонтальных углов</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="stddev_zenith_angle">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>390</y>
               <width>113</width>
               <height>20</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_8">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>390</y>
               <width>161</width>
               <height>16</height>
              </rect>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::NoContextMenu</enum>
             </property>
             <property name="text">
              <string>С.К.О вертикальных углов</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="stddev_distance">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>420</y>
               <width>113</width>
               <height>20</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_9">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>420</y>
               <width>161</width>
               <height>16</height>
              </rect>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::NoContextMenu</enum>
             </property>
             <property name="text">
              <string>С.К.О расстояний</string>
             </property>
            </widget>
            <widget class="Line" name="line_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>440</y>
               <width>381</width>
               <height>20</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
            <widget class="QLabel" name="label_19">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>460</y>
               <width>161</width>
               <height>16</height>
              </rect>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::NoContextMenu</enum>
             </property>
             <property name="text">
              <string>Алгоритм уравнивания</string>
             </property>
            </widget>
            <widget class="QComboBox" name="adjust_algo">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>460</y>
               <width>191</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QCheckBox" name="adjustEndNodes">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>490</y>
               <width>361</width>
               <height>17</height>
              </rect>
             </property>
             <property name="text">
              <string>Включать в уравнивание ситуационные точки</string>
             </property>
            </widget>
            <widget class="Line" name="line_3">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>510</y>
               <width>381</width>
               <height>20</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
            <widget class="QLabel" name="label_20">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>530</y>
               <width>141</width>
               <height>16</height>
              </rect>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::NoContextMenu</enum>
             </property>
             <property name="text">
              <string>Масштаб отображения</string>
             </property>
            </widget>
            <widget class="QComboBox" name="map_scale">
             <property name="geometry">
              <rect>
               <x>180</x>
               <y>530</y>
               <width>191</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="tab_stations">
            <attribute name="title">
             <string>Станции</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="margin">
              <number>4</number>
             </property>
             <item>
              <widget class="QSplitter" name="splitter_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                 <horstretch>2</horstretch>
                 <verstretch>2</verstretch>
                </sizepolicy>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <widget class="QTableWidget" name="StationsWidget"/>
               <widget class="QTreeWidget" name="ObservationsWidget">
                <column>
                 <property name="text">
                  <string notr="true">1</string>
                 </property>
                </column>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_points">
            <attribute name="title">
             <string>Точки</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="margin">
              <number>4</number>
             </property>
             <item>
              <widget class="QTableWidget" name="PointsWidget"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_adjust_points">
            <attribute name="title">
             <string>Уравненные</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="margin">
              <number>4</number>
             </property>
             <item>
              <widget class="QTableWidget" name="APointsWidget"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_listing">
            <attribute name="title">
             <string>Протокол</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="margin">
              <number>4</number>
             </property>
             <item>
              <widget class="QTextBrowser" name="textBrowser"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_viewports">
       <attribute name="title">
        <string>Схема хода</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_tv">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter_Viewports">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="handleWidth">
           <number>4</number>
          </property>
          <widget class="QSplitter" name="splitter_VP_list_props">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="handleWidth">
            <number>4</number>
           </property>
           <widget class="QTreeWidget" name="treeViewports">
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
           <widget class="QTableWidget" name="tableViewportsProperty"/>
          </widget>
          <widget class="QTabWidget" name="tabWidgetPrintViewports">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="tabPosition">
            <enum>QTabWidget::South</enum>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tabPWViewportEdit">
            <attribute name="title">
             <string>Размещение</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_VPE">
             <property name="margin">
              <number>4</number>
             </property>
            </layout>
           </widget>
           <widget class="QWidget" name="tapPWViewEdit">
            <attribute name="title">
             <string>Просмотр</string>
            </attribute>
            <layout class="QHBoxLayout" name="horizontalLayout_VE">
             <property name="margin">
              <number>4</number>
             </property>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Печатать</string>
     </property>
     <addaction name="printAdjustLog"/>
     <addaction name="printAdjustPoints"/>
     <addaction name="printSitPoints"/>
    </widget>
    <addaction name="NewNet"/>
    <addaction name="separator"/>
    <addaction name="NetOpen"/>
    <addaction name="NetClose"/>
    <addaction name="NetSave"/>
    <addaction name="NetSaveAs"/>
    <addaction name="separator"/>
    <addaction name="editBaseList"/>
    <addaction name="separator"/>
    <addaction name="menu_4"/>
    <addaction name="separator"/>
    <addaction name="Exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Сеть</string>
    </property>
    <widget class="QMenu" name="menu_7">
     <property name="title">
      <string>Экспорт</string>
     </property>
     <addaction name="actionExportTxt"/>
     <addaction name="actionExportCsv"/>
     <addaction name="actionExportDxf"/>
    </widget>
    <addaction name="AdjustNetwork"/>
    <addaction name="separator"/>
    <addaction name="menu_7"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="zoomIn"/>
    <addaction name="zoomOut"/>
    <addaction name="zoom100"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>Инструмент</string>
    </property>
    <addaction name="Import"/>
    <addaction name="separator"/>
    <addaction name="action_inportCOM"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_6"/>
   <addaction name="menu_3"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="NetOpen">
   <property name="text">
    <string>Открыть сеть</string>
   </property>
  </action>
  <action name="NetClose">
   <property name="text">
    <string>Закрыть сеть</string>
   </property>
  </action>
  <action name="NetSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="NetSaveAs">
   <property name="text">
    <string>Сохранить как ...</string>
   </property>
  </action>
  <action name="Exit">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="NewNet">
   <property name="text">
    <string>Новая сеть</string>
   </property>
  </action>
  <action name="AdjustNetwork">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Уравнять</string>
   </property>
  </action>
  <action name="zoomIn">
   <property name="text">
    <string>Ближе</string>
   </property>
  </action>
  <action name="zoomOut">
   <property name="text">
    <string>Дальше</string>
   </property>
  </action>
  <action name="zoom100">
   <property name="text">
    <string>100%</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Текст</string>
   </property>
  </action>
  <action name="actionExportTxt">
   <property name="text">
    <string>Текст (TXT)</string>
   </property>
  </action>
  <action name="actionExportCsv">
   <property name="text">
    <string>Текст (CSV)</string>
   </property>
  </action>
  <action name="actionExportDxf">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Автокад (DXF)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_inportCOM">
   <property name="text">
    <string>Загрузка через COM порт</string>
   </property>
  </action>
  <action name="Import">
   <property name="text">
    <string>Импорт измерений из файла ...</string>
   </property>
  </action>
  <action name="printAdjustLog">
   <property name="text">
    <string>Протокол уравнивания</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Загрузить пункты</string>
   </property>
  </action>
  <action name="editBaseList">
   <property name="text">
    <string>Каталог пунктов ...</string>
   </property>
  </action>
  <action name="printAdjustPoints">
   <property name="text">
    <string>Ведомость координат ПВО</string>
   </property>
  </action>
  <action name="printSitPoints">
   <property name="text">
    <string>Ведомость координат тахеометрии</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
